tools/check_volume_levels.m

```diff
@@ -251,3 +279,29 @@ function plot_levels(meas, tc, lm)
 		y(:,j) = filter(b, a, x(:,j));
 	end
 end
+
+% This function is copy of
+% sof/tools/test/audio/test_utils/multitone.m
+function x = multitone( fs, f, amp, tlength )
+	n = round(fs*tlength);
+	t = (0:n-1)/fs;
```

Comment:
can you please add space for the operation t = (0:n-1)/fs; like `t = (0:n-1) / fs;`